data_structures/binary_tree/morris_inorder_traversal.py

```diff
@@ -0,0 +1,168 @@
+"""
+Problem Statement: Given a binary perform an inorder traversal using Morris Inorder
+traversal algorithm. (Iterative version of Inorder traversal of tree)
+
+https://www.geeksforgeeks.org/inorder-tree-traversal-without-recursion-and-without-stack/
+"""
+
+
+class TreeNode:
+    """
+    Class representing a node in a binary tree.
+
+    Attributes:
+    -----------
+    value : int
+        The value stored at the node.
+    left : TreeNode
+        Pointer to the left child node (default is None).
+    right : TreeNode
+        Pointer to the right child node (default is None).
```

Comment:
```suggestion
    value: The value stored at the node.
    left: Pointer to the left child node (default is None).
    right: Pointer to the right child node (default is None).
```
By convention in this repo, we prefer not to include the types of the function params in the docstrings. This is because the functions themselves should already have type hints, and adding the types in the docstring means that, if we ever change the variables' types, we'd need to update the type hints in two separate places.

Could you make similar changes in the other docstrings as well?